src/features/commands.ts

```diff
@@ -198,13 +198,13 @@ async function dotnetRestoreAllProjects(server: OmniSharpServer, eventStream: Ev
 
 async function getProjectDescriptors(server: OmniSharpServer): Promise<protocol.ProjectDescriptor[]> {
     if (!server.isRunning()) {
-        return Promise.reject('OmniSharp server is not running.');
+        throw new Error('OmniSharp server is not running.');
```

Comment:
rejecting the promise causes the the stack to be lost in the jest output.  throwing an error ensures its included.